src/app.js

```diff
@@ -1,59 +1,55 @@
-export function app(app) {
-  var state = {}
+export function app(props) {
+  var state
   var actions = {}
   var events = {}
   var mixins = []
-  var view = app.view
-  var root = app.root || document.body
+  var view = props.view
+  var root = props.root || document.body
   var node
   var element
   var locked = false
-  var loaded = false
 
   for (var i = -1; i < mixins.length; i++) {
-    var mixin = mixins[i] ? mixins[i](emit) : app
+    props = mixins[i] ? mixins[i](emit) : props
 
-    Object.keys(mixin.events || []).map(function(key) {
-      events[key] = (events[key] || []).concat(mixin.events[key])
+    Object.keys(props.events || []).map(function(key) {
+      events[key] = (events[key] || []).concat(props.events[key])
     })
 
-    if (mixin.state != null) {
-      state = merge(state, mixin.state)
-    }
-
-    mixins = mixins.concat(mixin.mixins || [])
+    iterate(actions, props.actions)
 
-    initialize(actions, mixin.actions)
+    mixins = mixins.concat(props.mixins || [])
+    state = merge(state, props.state)
   }
 
-  node = hydrate((element = root.querySelector("[data-ssr]")), [].map)
+  schedule(
+    (node = hydrate((element = root.querySelector("[data-ssr]")), [].map))
+  )
 
-  repaint(emit("init"))
-
-  return emit
+  return emit("load")
```

Comment:
Why not : 
```jsx
return emit("load", emit)
```

To return emit by default for non breaking change here. And emit will be given to `load` as data to permit to return emit even if you are returning something else.

```jsx
loads(state, actions, emit) {
  return { actions, emit }
}
```